### PR TITLE
API-6698: document auto access

### PR DESCRIPTION
**Pull request by olek-p**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-5804-auto-access-rules)
- [Jira](https://livechatinc.atlassian.net/browse/API-6698)

## 📓 Description
Docs for the auto access rules feature: the rules can be managed via the new methods in configuration-api v3.3.
There was already an attempt to release the feature in the past, hence the "old" branch name.

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content
- New content in configuration v3.3: new section Auto access rules
- Changelog entry for v3.3
  
### Features (for the website)
- New feature

### CODE REVIEW

File: content/management/configuration-api/v3.3/index.mdx [hunk 1]
```diff
@@ -680,6 +680,237 @@ curl -X POST \
...
+
+|                |                                                                                  |
+| -------------- | -------------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.3/configuration/action/delete_auto_access`       |
+| Required scopes| `access_rules:rw`                                                                |
+
+
+| Parameter              | Required     | Data type | Notes                         |
+| ---------------------- | ------------ | --------- | ----------------------------- |
+| `id`                   | Yes          | `string`  | auto access ID          |
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:886]:
```suggestion
| `id`                   | Yes          | `string`  | Auto access ID          |
```

Comment (olek-p) [reply content/management/configuration-api/v3.3/index.mdx]: Fixed + some more docs generated

File: content/management/configuration-api/v3.3/index.mdx [hunk 2]
```diff
@@ -680,6 +680,291 @@ curl -X POST \
...
+|                |                                                                                  |
+| -------------- | -------------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.3/configuration/action/reorder_auto_access`      |
+| Required scopes| `access_rules:rw`                                                                |
+
+
+| Parameter              | Required     | Data type | Notes                                                                                                                     |
+| ---------------------- | ------------ | --------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `id`                   | Yes          | `string`  | ID of the auto access to move                                                                                             |
+| `next_id`              | No           | `string`  | ID of the auto access which should follow the moved auto access. Leave empty if the moved auto access should be the last. |
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:940]:
```suggestion
| `next_id`              | No           | `string`  | ID of the auto access that should follow the moved auto access. Leave empty if the moved auto access should be the last. |
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 3]
```diff
@@ -680,6 +680,291 @@ curl -X POST \
...
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Reorder Auto Access
+
+Moves an existing `auto access` data structure, specified by `id`, in front of another, specified by `next_id`.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:926]:
```suggestion
Moves an existing `auto access` data structure, specified by `id`, before another one, specified by `next_id`. 
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 4]
```diff
@@ -680,6 +680,291 @@ curl -X POST \
...
+</Section>
+
+<Section>
+
+<Text>
+
+### Reorder Auto Access
+
+Moves an existing `auto access` data structure, specified by `id`, in front of another, specified by `next_id`. 
+If `next_id` is empty, then the auto access will be placed at the end.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:927]:
```suggestion
If `next_id` is empty, then the auto access will be placed at the end of the list.
```

File: content/management/configuration-api/v3.3/index.mdx (hunk 2 above)
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:940]: (...) should be the last one on the list.
(...) should be at the end of the list.
(...) should be the last one.

It just feels like this sentence is missing something.

Comment (olek-p) [reply content/management/configuration-api/v3.3/index.mdx]: Rewritten using `should be the last one on the list`

File: content/management/configuration-api/v3.3/index.mdx [hunk 5]
```diff
@@ -680,6 +680,291 @@ curl -X POST \
...
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/reorder_auto_access \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "id": "pqi8oasdjahuakndw9nsad9na"
```
Comment (lpmichalik) [inline content/management/configuration-api/v3.3/index.mdx:958]: Maybe example should actually include the next_id?

Comment (ale-kacperczyk) [reply content/management/configuration-api/v3.3/index.mdx:958]: Good idea 👆🏻

Comment (olek-p) [reply content/management/configuration-api/v3.3/index.mdx:958]: In each of the methods the `next_id` param is optional, and I used it in other examples, so I figured I'd leave one out ;) But no problem, I can add it here too.

Comment (olek-p) [reply content/management/configuration-api/v3.3/index.mdx]: Fixed nao

**Pull request merged**